packages/commonwealth/client/scripts/helpers/ContractHelpers/Contest.ts

```diff
@@ -0,0 +1,125 @@
+import { AppError } from '@hicommonwealth/core';
+import { ContestAbi } from './Abi/ContestAbi';
+import ContractBase from './ContractBase';
+import NamespaceFactory from './NamespaceFactory';
+
+class Contest extends ContractBase {
+  namespaceFactoryAddress: string;
+  namespaceFactory: NamespaceFactory;
+
+  constructor(contractAddress: string, factoryAddress: string, rpc: string) {
+    super(contractAddress, ContestAbi, rpc);
+    this.namespaceFactoryAddress = factoryAddress;
+  }
+
+  async initialize(withWallet: boolean = false): Promise<void> {
+    await super.initialize(withWallet);
+    this.namespaceFactory = new NamespaceFactory(
+      this.namespaceFactoryAddress,
+      this.rpc,
+    );
+    await this.namespaceFactory.initialize();
+  }
+
+  /**
+   * deploys a new recurring contest and configures the feeManager to divert feeShare of
+   * stake fees to new contest
+   * @param namespaceName namespace name to configure for
+   * @param contestInterval the recurrng contest interval in ms
+   * @param winnerShares the percent of each respective winning places prize(ie 5 = 5%)
```

Comment:
just to confirm that this is it on the UI:

<img width="451" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14819225/e23750ab-9424-4c3e-86b2-a21e982bc612">

on platform it's called `payout_structure` and it [lives here](https://github.com/hicommonwealth/commonwealth/blob/a2de535b7415e9ec597b39db1caf9bbc3d29a80d/libs/core/src/schemas/entities.schemas.ts#L316)

Reply:
correct